system/lib/compiler-rt/lib/lsan/lsan_interceptors.cpp

```diff
@@ -562,15 +562,14 @@ void InitializeInterceptors() {
   LSAN_MAYBE_INTERCEPT_PTHREAD_ATFORK;
 
   LSAN_MAYBE_INTERCEPT_STRERROR;
+#endif  // !SANITIZER_FUCHSIA && !SANITIZER_EMSCRIPTEN
```

Comment:
is the movement of the fuscia ifdef here a divergence from upstream?

Reply:
(otherwise lgtm)

Reply:
Oh yes, I hadn't realized I had effect fuchsia here.  Will follow up.